bin/create-layer-templates

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const yargs = require('yargs/yargs');
+const { hideBin } = require('yargs/helpers');
+const { createLayerTemplate, createVariantTemplate } = require('../dist/main');
+
+const argv = yargs(hideBin(process.argv))
```

Comment:
Can we add some basic validation where we try opening each of the directories/files and output a message if any fails?